equivalence.red

```diff
@@ -0,0 +1,14 @@
+import path
+
+let IsContr (C : type) : type =
+  (c : _) × (c' : _) →
+    Path C c' c
```

Comment:
BTW this is actually `(c : _) × ((c' : _) → Path C c' c)` which is different from other proof assistants...

Reply:
Fwiw I don't think there's any ambiguity compared to other proof assistants, because the second part of a sigma type doesn't have a named binder. So even in a hypothetical version of Agda that has the `(x : A) * B` notation (or just Nuprl, which does have it), you would write it this way and it wouldn't be ambiguous.

Reply:
Binders are blurring my point. I think `A × B → C` is usually parsed as `(A × B) → C` not `A × (B → C)`.

Reply:
Well, then I should rephrase my complaint as that I think it would be more consistent to have `A → B` be syntactically identical to `(_ : A) → B` and also `A × B` to `(_ : A) × B`?

Reply:
If we rephrase the complaint in that way, then I don't think I agree with it :stuck_out_tongue_winking_eye:

Reply:
Hmm. In your opinion, should `(a : A) * (b : B) -> C -> D` be parsed as `(a : A) * ((b : B) -> (C -> D))` or ``(a : A) * ((b : B) -> C) -> D``?

Reply:
It has to be parsed as the former, because the function arrow associates to the right.